sql-native/src/main/java/org/ballerinalang/sql/utils/ColumnDefinition.java

```diff
@@ -32,7 +32,7 @@
     private final Type ballerinaType;
     private final boolean isNullable;
 
-    ColumnDefinition(String columnName, String ballerinaFieldName, int sqlType, String sqlName,
+    public ColumnDefinition(String columnName, String ballerinaFieldName, int sqlType, String sqlName,
```

Comment:
We cant change visibility just because we want to test this.

Reply:
@niveathika, Either we can change the ColumnDefinition constructor visibility to `protected` or move the test class to the same package as `ColumnDefinition` in order to access the constructor. What is the best way to do that ?

Reply:
IMO best way to not add those tests. Lets check and see if we can improve coverage without this.

Reply:
Visibility changed in https://github.com/ballerina-platform/module-ballerina-sql/pull/291/commits/90fbf662f91b2bd829e555ad600154b489bad617